docs/sources/tempo/configuration/grafana-alloy/tail-sampling/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Sampling
+menuTItle: Sampling
+description: Use sampling to optimize sampling decisions
+weight:
+aliases:
+- /docs/tempo/grafana-alloy/tail-based-sampling
+- ../tail-based-sampling/
+---
+
+# Sampling
+
+Grafana Tempo provides an inexpensive solution that aims to ingest and store the traces that provide maximum observability across your application estate.
+However, sometimes constraints mean that storing all of your traces is not desirable, for example runtime or egress traffic related costs.
+There are a number of ways to lower trace volume, including varying sampling strategies.
+
+Sampling is the process of determining which traces to store (in Tempo or Grafana Cloud Traces) and which to discard. Sampling comes in two different strategy types: head and tail sampling.
```

Comment:
again - i think if the intention of the doc is to provide generic conceptual docs on sampling - then we shouldn't refer to it as being tempo or traces specific? We'd split the generic conceptual stuff out and then have product-specific docs living in each of our areas for the details on each product? Or you rename the doc to Sampling in Traces?